### ISSUE TITLE
Calender-Page scrolling area is way too long

### ISSUE THREAD

**Issue by FabianSimonGross** [neuland-ingolstadt/neuland.app#165]
You basically can scroll into Oblivion on /calendar which is not a good user experience.

**Comment by alexhorn** [comment]
This is because our implementation of the tab control makes all tabs as long at the longest tab. Since react-swipeable-views seems to be deprecated anyway, we should consider replacing it.

**Comment by M4GNV5** [comment]
A simple fix would probably be to make all vies empty except the currently active one, should be an easy fix.
Is there a way to get the current active tab from a `SwipeableTabs` instance?

**Comment by alexhorn** [comment]
@M4GNV5 During the swipe motion two tabs have to be rendered at once, so I'm not sure that will work

**Comment by M4GNV5** [comment]
With the new split between `/calendar` and `/events` this problem is way less present. Do we even care enough to keep this issue open?

**Comment by Robert27** [comment]
You are right. I think we can close the issue as not implemented or mark it as unimportant  for now.
